### ISSUE TITLE
Retrain ra-gcn model + st-gcn baseline (spatial configuration)

### ISSUE THREAD

**Issue by thancaocuong** [yfsong0709/RA-GCNv1#5]
Hi, thanks for awsome implementation. As you know in st-gcn has 3 Partitioning. I find that you only use Distance partitioning and gain good result. But I'm curious about how can I use Spatial Configuration for graph parttitioning? I tried to use pretrained from st-gcn but size mismatch for data_bn.weight because of your data is (9,300,25,2) => data_bn.weight size is 450 (in st-gcn is 75).
can you tell me how to produce baseline model? 
```
size mismatch for data_bn.weight: copying a param with shape torch.Size([75]) from checkpoint, the shape in current model is torch.Size([450])
```

**Comment by thancaocuong** [comment]
@yfsong0709  please

**Comment by yfsong0709** [comment]
@thancaocuong Thanks for using my code... 450 / 75 = 6... I guess, maybe you are using the Yan's ST-GCN model. There are a little bit different between the two models. My data_bn is defined as nn.BatchNorm1d(C_enrich * V * M), while Yan's ST-GCN is defined as nn.BatchNorm1d(C * V), where V=25, M=2, C=3 and C_enrich=9. You can try to modify this BN layer and run again.

**Comment by thancaocuong** [comment]
Thank you, But can I train st-gcn of Yan's ST_GCN to get pretrained model or modify your source code to use Yan's ST-GCN pretrained?.

**Comment by yfsong0709** [comment]
If you want to use Yan's st-gcn pretrained model, you can try to modify my code. This look good in theory, but may be a little tedious to implement.

**Comment by thancaocuong** [comment]
@yfsong0709 can I use your code to produce baseline model?

**Comment by yfsong0709** [comment]
It is easy to produce baseline model by using my code. You only need to use the 'ST_GCN' class in 'nets.py' as the training model. Some details need to be modified, but it is not very difficult.

**Comment by thancaocuong** [comment]
oh thank you. I will try to do
